uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -337,6 +359,59 @@ =head2 Methods
             . "inserted into the database \n";
 spool($message,'N', $notify_notsummary);
 
+################################################################
+############### Email last completion message ##################
+################################################################
+my @inserted_files = getInsertedFileNamesUsingUploadID($upload_id);
```

Comment:
The function getInsertedFileNamesUsingUploadID() should be in ImagingUpload.pm and this line replaced by
```suggestion
my @inserted_files = $imaging_upload->getInsertedFileNamesUsingUploadID($upload_id);
```